### PR TITLE
Consolidate under Operations

### PR DESCRIPTION
**Pull request by joe-elliott**
**What this PR does**:
- Moves deployment and architecture under operations
- Uses aliases to redirect old pages to new
- Minor improvements to architecture page

### CODE REVIEW

File: docs/tempo/website/operations/_index.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ---
-title: Operations
+title: Deployment and Operations
```
Comment (dgzlopes) [inline docs/tempo/website/operations/_index.md:2]: I kinda prefer having `Operations` as the title. (for me) Deployment is just a subsection inside Operations.

In the same way, that on the repo we have the Deployment stuff inside the [Operations](https://github.com/grafana/tempo/tree/main/operations) folder.

File: docs/tempo/website/operations/_index.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 weight: 300
 ---
 
-Operations for Tempo include
+- [Deployment](deployment)
 - [Monitoring](monitoring)
 - [Tempo CLI](tempo_cli)
+- [Architecture](architecture)
```
Comment (dgzlopes) [inline docs/tempo/website/operations/_index.md:9]: I liked the idea of having Architecture on the top layer (like a page, we don't need a folder for it). 

No strong feelings about putting this inside Operations, just commenting on it :smile:

**Review by dgzlopes — COMMENTED** [review]
Left some comments! Nothing major :)

File: docs/tempo/website/operations/_index.md (hunk 1 above)
Comment (joe-elliott) [reply docs/tempo/website/operations/_index.md:2]: So the intent of this was to just remove these single page top level items to reduce confusion and I felt like both Architecture and Deployment made sense under Operations. 

The reason for the title change to "Deployment and Operations" is I think that a common question coming to the docs would be "How do I deploy this?" and I wanted it to be very clear where to go in that case.

File: docs/tempo/website/operations/architecture.md [hunk 3]
```diff
@@ -24,13 +27,17 @@ the [Grafana Agent](https://github.com/grafana/agent) uses the otlp exporter/rec
...
 <bucketname> / <tenantID> / <blockID> / <meta.json>
-.                                     / <bloom>
-.                                     / <index>
-.                                     / <data>
+                                      / <index>
+                                      / <data>
+                                      / <bloom_0>
+                                      / <bloom_1>
+                                        ...
+                                      / <bloom_n>
```
Comment (annanay25) [inline docs/tempo/website/operations/architecture.md:40]: We need another small change in the Querier section of this page. 

```
It begins by querying the ingesters to see if the id is currently stored there, if not it proceeds to use the bloom and indexes to find the trace in the storage backend.
```

This is no longer done sequentially, we can simplify this sentence.. "it searches the ingester and blocks for traces"

File: docs/tempo/website/_index.md [hunk 4]
```diff
@@ -10,13 +10,10 @@ aliases:
 
 Grafana Tempo is an open source, easy-to-use and high-volume distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Prometheus, and Loki. To learn more about Grafana, refer to [What is Grafana?](https://grafana.com/docs/grafana/latest/getting-started/)
 
-
 - [Getting Started](getting-started/)
 - [Configuration](configuration/)
-- [Deployment](deployment/)
 - [Operations](operations/)
```
Comment (annanay25) [inline docs/tempo/website/_index.md:15]:
```suggestion
- [Deployment and Operations](deployment-and-operations/)
```

IIUC, the link will work only this way^ ?

**Review by annanay25 — APPROVED** [review]

**Pull request merged**